### DISCUSSION TITLE
Roadmap to the official implementation

### DISCUSSION THREAD

**Discussion by fabiogouw** [testcontainers/testcontainers-dotnet#442] [General]
In the last Thoughtworks tech radar, [Testcontainers was put int the adopt ring](https://www.thoughtworks.com/radar/languages-and-frameworks?blipid=201911027). This is great, they've commented that the library is ported to multiple languages, but we know that the official .NET version is not mature as Hofmeinster's.
Is there any roadmap, any plans to incorporate all work done here in the official implementation?

**Comment by HofmeisterAn** [comment]
It's great to see that Testcontainers continues to grow and that it's considered as a default option for creating a reliable environment for running tests 💪. Especially that it's part of the Technology Radar.

An official .NET version would be huge. There are so many benefits and synergies bundling all the work together. There were some efforts in the past, but unfortunately, without a lot of success.

Anyway, I'm still happy to share my knowledge and work. If there's anything I can do, I'm happy to jump in and contribute.

**Reply by fabiogouw** [reply]
I've noticed that your version of testcontainers has been moved as the official repo, that's great! Congrats!
